projects/FirstProjects/FirstProject/src/com/valentin/Main.java

```diff
@@ -0,0 +1,131 @@
+                    case 2:
+                        greatestCommonDiv();
+                        Thread.sleep(1000);
+                        System.out.println("\r\n");
+                        break;
+                    case 3:
+                        int[] tempArr = enterArr();
+                        commonArr = Arrays.copyOf(tempArr, tempArr.length);
+                        arrClass.printArr(commonArr);
+                        Thread.sleep(1000);
+                        System.out.println("\r\n");
+                        break;
+                    case 4:
+                        System.out.println("The min element is: " + arrClass.getMinElement(commonArr));
+                        Thread.sleep(1000);
+                        System.out.println("\r\n");
+                        break;
+                    case 5:
+                        System.out.println("The sum is: " + arrClass.getSum(commonArr));
+                        Thread.sleep(1000);
+                        System.out.println("\r\n");
+                        break;
+                    case 6:
+                        arrClass.printArr(commonArr);
+                        int[] arrSorted = quickSort.quickSort(commonArr);
+                        System.out.println("The sorted arr is:");
+                        arrClass.printArr(arrSorted);
+                        System.out.println("\r\n");
+                        Thread.sleep(1000);
+                        break;
+                    case 7:
+                        System.out.println("The reversed arr: ");
+                        arrClass.printArr(reverseArray.reverseArray(commonArr));
+                        System.out.println("\r\n");
+                        Thread.sleep(1000);
+                        break;
+                    case 8:
+                        return;
+                }
+
+
+            }catch (Exception e){
+                e.printStackTrace();
+                continue;
+            }
+        }
+    }
+
+    public static void biggestCommonDiv() {
+        //Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a: ");
+        int a = scanner.nextInt();
+        System.out.println("Enter b:");
+        int b = scanner.nextInt();
+        while (a!=b){
+            if (a>b){
+                a=a-b;
+            }else
+            {
+                b=b-a;
+            }
+        }
+        System.out.println("The BCD is: " + a);
+    }
+
+
+    /**
+     *
+     */
+    public static void greatestCommonDiv(){
+
+        System.out.println("Enter a: ");
+        int a = scanner.nextInt();
+        System.out.println("Enter b:");
+        int b = scanner.nextInt();
+        while (b!=0){
+            int c=a%b;
+            a=b;
+            b=c;
+
+        }
+        System.out.println("The GCD is: "+a);
+    }
+
+    /**
+     *
+     * @return
+     */
+    public static int[] enterArr(){
+        System.out.println("How many numbers?");
```

Comment:
The production code shouldn't use system out or system in inless it's purpose is to print something in the console such as:

```java
class ConsoleDisplay implements Display {
   public void printText(String text) {
      System.out.printf("Text: %s\n", text); 
   }
}
```